apps/myaccount/src/components/authentication/sign-in.tsx

```diff
@@ -44,14 +46,15 @@ const SignIn: FunctionComponent<RouteComponentProps> = (
     const loginSuccessRedirect = () => {
 
         // TODO: Fix the auth callback isssue here.
-        /* const AuthenticationCallbackUrl = AuthenticateUtils.getAuthenticationCallbackUrl();
+        const AuthenticationCallbackUrl = AuthenticateUtils.getAuthenticationCallbackUrl(
```

Comment:
Have you tested the flow in tenant mode?

If it works fine in tenant mode as well, we can remove the `TODO` comment.

Reply:
I will test it in the tenant mode and see if this works fine.

Reply:
I tested it and it works fine.